src/main/kotlin/de/bund/digitalservice/useid/statics/WidgetController.kt

```diff
@@ -9,4 +9,7 @@ import reactor.core.publisher.Mono
 class WidgetController {
     @GetMapping("/widget")
     fun widget(model: Model): Mono<String> = Mono.just("widget")
+
+    @GetMapping("/incompatible")
```

Comment:
still unsure with the naming. `/not-supported` ?

Reply:
/not-supported sounds a bit smoother. But really unsure about this as well